packages/manager/modules/cloud-connect/src/index.js

```diff
@@ -0,0 +1,51 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'oclazyload';
+import 'ovh-api-services';
+import moment from 'moment';
+import ngOvhCloudUniverseComponents from '@ovh-ux/ng-ovh-cloud-universe-components';
+
+import './style.scss';
+
+const moduleName = 'ovhCloudConnectLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ovhManagerCore',
+    'pascalprecht.translate',
+    'ui.router',
+    'oc.lazyLoad',
+    'ovh-api-services',
+    ngOvhCloudUniverseComponents,
+  ])
+  .config(
+    /* @ngInject */ ($stateProvider) => {
+      $stateProvider.state('cloud-connect.**', {
+        url: '/cloud-connect/:ovhCloudConnectId',
+        lazyLoad: ($transition$) => {
+          const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+          return import('./cloud-connect.module').then((mod) =>
+            $ocLazyLoad.inject(mod.default || mod),
+          );
+        },
+      });
+    },
+  )
+  .run(
+    /* @ngInject */ ($translate) => {
+      let lang = $translate.use();
+      if (['en_GB', 'es_US', 'fr_CA'].includes(lang)) {
+        lang = lang.toLowerCase().replace('_', '-');
+      } else {
+        [lang] = lang.split('_');
+      }
+      return import(`script-loader!moment/locale/${lang}.js`).then(() =>
+        moment.locale(lang),
+      );
+    },
+  );
```

Comment:
This is not the responsibility of the module to load moment locales.

Reply:
This is run in `manager-core` right ? Sometimes moment is not set to `locale`. Am i missing anything ?

Reply:
`script-loader` should only be used in apps since there are processed with webpack (here it's ok because the module is also processed by webpack due to the import in web but it won't work if we were building it with rollup)
Maybe the moment instance used is a new one and not the one set globally so it will use the default one

Reply:
`moment` is already configured in `dedicated` : https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/app.js#L304-L318

`moment` is global so we should configure it globally, if we update it in multiple module we could have side effect.

Reply:
Have added global moment and removed local instance which was getting instantiated.